tests/launcher/test_service.py

```diff
@@ -305,112 +304,210 @@ def test_service_get_jobs_from_database(self, db_session) -> None:
     @pytest.mark.parametrize(
         "config, solver, expected",
         [
+            # pytest.param(
```

Comment:
In general, we should either completely delete tests or make them pass (otherwise we'll end up with too many commented stuff :) ) :
here it seems that the commented out test actually pass, so I think we can re-activate them